src/sliding-tab/ExNavigationSlidingTab.js

```diff
@@ -42,6 +42,7 @@ type Props = {
   position: "top" | "bottom",
   pressColor: ?string,
   renderBefore: () => ?ReactElement<any>,
+  renderTabBar: (props: Object) => React.Element<{}>,
```

Comment:
The signature should be `(props: TabBarProps) => ?React.Element<any>` (add type for the props and allow returning null)